lib/gcov/ukgcov.c

```diff
@@ -0,0 +1,105 @@
+/* SPDX-License-Identifier: MIT */
+/* Copyright © 2005-2020 Rich Felker, et al.
```

Comment:
Where is this license and copyright derived from? Bellow you (correctly) reference https://gcc.gnu.org/onlinedocs/gcc/Freestanding-Environments.html, but I don't see anything relevant there.